src/main/resources/helptexts/typesDCMI.html

```diff
@@ -1,5 +1,18 @@
-<h1>Test header</h1>
-<p>Lorem ipsum dolor sit amet, consectetur adipiscing elit. Pellentesque semper nulla eu placerat consectetur.</p>
-<p>Sed dignissim dapibus venenatis. Sed rhoncus.</p>
-<h2>Subheader2</h2>
-<p>Maecenas luctus metus eleifend, pulvinar mi ac, pellentesque nibh. Sed feugiat, augue.</p>
+<h1>TType</h1>
+<p>General characteristic of the dataset deposited.</p>
+<p>The field ‘Type' can be used to indicate the form of the dataset in general terms. You can opt
+    for both an archaeological description of the contents and a technical description of the
+    contents. If you opt for a technical description, it is recommended that you do this in
+    conformity with the international thesaurus called ‘DCMI resource type'. The default value will
+    then be ‘Dataset'.</p>
+<p>Click the <i class="fas fa-plus-square"></i> button behind a field to add an additional field.
+</p>
+<h2>Examples</h2>
+<ul>
```

Comment:
only keep the ones that are in DCMI, so Dataset, Text, Image, and delete the others